dart/pom.xml

```diff
@@ -35,6 +35,18 @@
       <artifactId>compile-testing</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.parceler</groupId>
+      <artifactId>parceler-api</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+
+    <dependency>
+        <groupId>com.squareup</groupId>
+        <artifactId>javapoet</artifactId>
```

Comment:
@johncarl81 , yes Dart as some class used at runtime. Namely, users must call `Dart.inject(context)` to invoke that has been generated during annotation processing.

@f2prateek , the intent builder's code generation is much simpler with this dependency. 
We could remove it for sure, but we would loose quality and maintenance comfort.
We can try the provided scope, it should work according to http://docs.codehaus.org/display/MAVENUSER/Dependency+Scopes

But maybe a better solution would be to separate runtime and compiler artifacts. Btw, @f2prateek , I thought you relied on proguard to remove the Dart annotation processor from final apk. And the same reasoning would have applied to any dependency here. But how does Dart annotation processor avoid getting included in final apk and not this dependency, so ?